python/lsst/ts/hexrotcomm/command_telemetry_client.py

```diff
@@ -131,6 +132,11 @@ def __init__(
         telemetry_callback,
         connect_timeout=10,
     ):
+        for arg_name in ("connect_callback", "config_callback", "telemetry_callback"):
+            arg_value = locals()[arg_name]
+            if not inspect.iscoroutinefunction(arg_value):
+                raise TypeError(f"{arg_name} must be a coroutine")
```

Comment:
Please consider to add the doc string of `TypeError` to `Raises` section of function description.